src/server/dash-server.ts

```diff
@@ -14,42 +15,74 @@ import {getRouter as getUpdatesRouter} from './apis/updates';
 import {enforceHTTPS} from './utils/enforce-https';
 import {requireLogin} from './utils/require-login';
 
+const STATIC_EXT = ['html'];
+
 export class DashServer {
   private app: express.Express;
   private server: Server|null;
 
   constructor() {
     this.server = null;
     const app = express();
-    const litPath = path.dirname(require.resolve('lit-html'));
 
     if (process.env.NODE_ENV === 'production') {
       app.use(enforceHTTPS);
     }
 
+    // Setup common middleware
     app.use(cookieParser());
-    app.use('/node_modules/lit-html', express.static(litPath));
-    app.use(express.static(
-        path.join(__dirname, '..', 'client', 'public'),
-        {extensions: ['html', 'mjs']}));
-    app.use(express.static(path.join(__dirname, '..', 'sw')));
-    app.use('/api/login/', bodyParser.text(), getLoginRouter());
-    app.use('/api/webhook/', bodyParser.json(), getGitHubHookRouter());
-
-    // Require Login for all endpoints used after this middleware
+    app.use(bodyParser.text());
+    app.use(bodyParser.json());
```

Comment:
These don't fail if they're not JSON?

Reply:
No - it requires the content type header to be set correctly.